subtests/docker_cli/build/build.py

```diff
@@ -208,22 +209,26 @@ def parse_postprocess_commands(self, build_def):
         self.logdebug("Postprocessing commands: %s", commands)
         return result
 
-    @staticmethod
-    def basic_postprocess(build_def, command, parameter):
+    def basic_postprocess(self, build_def, command, parameter):
         del parameter  # not used
         if command == 'positive':
-            # Verify zero exit status + healthy output
-            if OutputGood(build_def.dockercmd.cmdresult,
-                          ignore_error=True):
+            # Verify zero exit status and healthy output
+            opg = OutputGood(build_def.dockercmd.cmdresult,
+                             ignore_error=True)
+            if opg:
                 return build_def.dockercmd.cmdresult.exit_status == 0
             else:
+                self.loginfo(str(opg))
                 return False
         elif command == 'negative':
-            # Verify non-zero exit status + healthy output
-            if OutputGood(build_def.dockercmd.cmdresult,
-                          ignore_error=True,
-                          skip=['error_check']):
+            # Verify non-zero exit status and no panics
+            opb = OutputNotBad(build_def.dockercmd.cmdresult,
```

Comment:
I'm really uncomfortable with the name `opb`, especially when contrasted to `opg`. I read opb as 'output bad', not 'output NOT bad'. A casual reader may make the same assumption. How about just renaming both opg and opb to `ok`?